opentelemetry/proto/trace/v1/trace.proto

```diff
@@ -178,7 +178,7 @@ message Span {
   // is the time when the server's application handler starts running.
   // Value is UNIX Epoch time in nanoseconds since 00:00:00 UTC on 1 January 1970.
   //
-  // This field is semantically required and it is expected that end_time >= start_time.
+  // This field is semantically required and it is expected that end_time <= start_time.
```

Comment:
I think the current wording is correct.

Reply:
Oh duh sorry I misread and got confused because the order is the same in the docstring below 🤦🏻